### DISCUSSION TITLE
Publishing build question

### DISCUSSION THREAD

**Discussion by xalixilax** [Kira-NT/mc-publish#27] [Q&A] [answered]
Hello, I'm currently trying to publish on Curseforge and Mordrinth but I'm having trouble with the primary-file. It might be obvious but it's the first time I'm working with actions and I'm not sure how to to use the previously build texture pack zip from Packsquash. 

I've added the link to my workflow in case you want to check it. 

https://github.com/xalixilax/xali-s-Enhanced-Vanilla-16x/blob/main/.github/workflows/publish.yml
https://github.com/ComunidadAylas/PackSquash-action

Thanks for your help!

**Comment by Kira-NT** [comment, marked as answer]
You need to understand what's the state of a machine executing your action. As you can clearly see, [this job](https://github.com/xalixilax/xali-s-Enhanced-Vanilla-16x/runs/8207321649?check_suite_focus=true) did not execute anything related to producing a required file. What's the solution? Don't turn a step of your job into a separate job, that's all.

Also, you need to specify

```yaml
loaders: minecraft
```

in order for you configuration to work, because `mc-publish` does not know what resource packs are at the moment.

And, as a side note, you do not need `softprops/action-gh-release`, `mc-publish` can create releases by itself.
